packages/o2-blocks/babel.config.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
```

Comment:
The babel plugins here would make a nice babel config shard.

Reply:
Ah, good point!